assets/tests/components/routeLadder.test.tsx

```diff
@@ -655,7 +655,62 @@ describe("routeLadder", () => {
     expect(tree).toMatchSnapshot()
   })
 
-  test("renders alert icon on a route ladder with an active detour", () => {
+  test("renders active detours in dropdown that were made in skate", async () => {
+    jest.mocked(getTestGroups).mockReturnValue([TestGroups.DetoursPilot])
+
+    const route: Route = routeFactory.build({
+      id: "28",
+      name: "28",
+    })
+    const timepoints = [
+      { id: "MATPN", name: "MATPN Name" },
+      { id: "WELLH", name: "WELLH Name" },
+      { id: "MORTN", name: "MORTN Name" },
+    ]
+    const skateDetours = {
+      "1": {
+        id: 1,
+        route: "28",
+        direction: "Inbound",
+        name: "",
+        intersection: "Main St @ South St",
+        updatedAt: 1,
+      },
+      "2": {
+        id: 2,
+        route: "28",
+        direction: "Outbound",
+        name: "",
+        intersection: "Main St @ South St",
+        updatedAt: 2,
+      },
```

Comment:
Can you use the factory to generate these, in case we change the structure of `SimpleDetour` later?

Reply:
Oh great! https://github.com/mbta/skate/pull/2915/commits/b381051b2fe2fce7848a45303217c9a924157c6e